File: 1635-NumberOfGoodPairs/1635-NumberOfGoodPairs.py
# Last updated: 1/8/2026, 3:32:55 PM
class Solution(object):
    def numIdenticalPairs(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        l=[]
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if (nums[i]==nums[j]):
                    l.append((i,j))
        return len(l)
        

File: 1635-NumberOfGoodPairs/1635-NumberOfGoodPairs.cpp
// Last updated: 1/8/2026, 3:32:50 PM
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int s=nums.size();
        int sum=0;
        for(int i=0;i<s;i++){
            for(int j=i+1;j<s;j++){
                if(nums[i]==nums[j]){
                    sum+=1;
                }
            }
        }
        return sum;
    }
};